checka11y.css

```diff
@@ -10,6 +10,19 @@
   --checka11y-font: 'Comic Sans MS', cursive !important;
 }
 
+/**
+* HTML
+*/
+
+html:not([lang]) body{
+  border: var(--checka11y-error-border-width, 5px) solid var(--checka11y-error-color, red) !important;
+}
+
+html:not([lang]) body::after{
+  content: "ERROR: html has no lang attribute. " !important;
```

Comment:
Sorry for being picky, could these be single quotes? I may consider configuring stylelint to the project, what do you think?

Reply:
Not a problem at all. I'm pushed an update for this